pkg/lib/csv/csv_test.go

```diff
@@ -0,0 +1,44 @@
+package csv
+
+import (
+	"testing"
+)
```

Comment:
for testing its better to use "github.com/stretchr/testify/assert" for asserting stuff

Reply:
It's not in your `go.mod` but your testing utils are sufficient (`AssertMapsEqual` doesn't support maps of slices though)